fmriprep/config.py

```diff
@@ -244,6 +244,8 @@ class nipype(_Config):
 
     crashfile_format = 'txt'
     """The file format for crashfiles, either text or pickle."""
+    disable_telemetry = bool(os.getenv("NIPYPE_NO_ET") is None)
```

Comment:
having a config for this is a bad idea - this is one of those options we don't want to reuse when reusing the config file. Please check mgxd/fmriprep#3 for an alternative.